### PR TITLE
Fix wrong "subversion" variable for job and batch name

### PR DESCRIPTION
**Pull request by friquette**
## Changelog Description
The code for the "subversion" variable was wrong, this should fix it.

## Testing notes:
Send a Maya job on Deadline with subversion and without subversion in the workfile name.

### CODE REVIEW

**Review by BenSouchet — CHANGES_REQUESTED** [review]

File: openpype/modules/deadline/utils.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def set_custom_deadline_name(instance, filename, setting):
     subversion = basename.split("_")[-1]
     version = "v" + str(instance.data.get("version")).zfill(3)
 
-    if subversion == version:
+    if re.match(r'^_v[0-9]{3}$', subversion):
```
[the remark this answers is not in the document]
Comment (friquette) [reply openpype/modules/deadline/utils.py]: fixed

**Comment by BenSouchet** [comment]
@friquette : Just to understand why checking the subversion part of the filename and if there is one setting the subversion to an empty string ?

**Comment by friquette** [comment]
> @friquette : Just to understand why checking the subversion part of the filename and if there is one setting the subversion to an empty string ?

I did this to avoid having two times the version in the job name if there is no subversion

**Comment by BenSouchet** [comment]
But your current code is `IF there is a subversion, remove it`.
This differ from `IF there is NOT a subversion, remove it`.

Do you have a screenshot of the wrong behavior occurring, when you doesn't want the subversion field.

**Comment by friquette** [comment]
> But your current code is `IF there is a subversion, remove it`. This differ from `IF there is NOT a subversion, remove it`.
> 
> Do you have a screenshot of the wrong behavior occurring, when you doesn't want the subversion field.

No, it says, ```IF the subversion looks like a version, remove it```
This was why the job name in the ticket had the version at the end though in the settings it was the subversion at the end

**Comment by BenSouchet** [comment]
> > But your current code is `IF there is a subversion, remove it`. This differ from `IF there is NOT a subversion, remove it`.
> > Do you have a screenshot of the wrong behavior occurring, when you doesn't want the subversion field.
> 
> No, it says, `IF the subversion looks like a version, remove it` This was why the job name in the ticket had the version at the end though in the settings it was the subversion at the end

Okay, I misunderstood the purpose of the subversion attribute, after looking up, it's a suffix/comment to differentiate or add info to a version.

**Comment by BenSouchet** [comment]
Merged in `3.16.9-quad-1.1.0`, will be available when we release it this week.

**Pull request closed without merging**